src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -103,7 +105,7 @@ private ItemCrafting ItemLevelUpgradeCrafting(byte craftingNumber, byte targetLe
         item.MaximumAmount = 1;
         item.MinimumItemLevel = (byte)(targetLevel - 1);
         item.MaximumItemLevel = item.MinimumItemLevel;
-        item.FailResult = MixResult.DowngradedTo0;
+        item.FailResult = MixResult.Disappear;
```

Comment:
If you don't remember that you've lost your item upon failed upgrading in CM, you were not much of a CM gambler :sweat_smile: https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L1390

Reply:
maybe this is when using 'Talisman of Chaos Assembly' (TOCA), where the item is just downgraded instead of disappearing. might need to add a condition here for TOCA ?

Reply:
There are no Cash Shop items implemented yet :)